### PR TITLE
deprecate character and numeric S3 dispatches

### PR DESCRIPTION
**Pull request by seth127**
implementing https://github.com/metrumresearchgroup/rbabylon/issues/173

### CODE REVIEW

File: R/copy-model-from.R [hunk 1]
```diff
@@ -4,19 +4,36 @@
...
-#' @param .add_tags Character vector with any new tags(s) to be added to `{.new_model}.yaml`
-#' @param .inherit_tags If `FALSE`, the default, new model will only have any tags passed to `.add_tags` argument. If `TRUE` inherit any tags from `.parent_mod`, with any tags passed to `.add_tags` appended.
-#' @param .update_model_file If `TRUE`, the default, update the newly created model file with new description and name.
-#' For a NONMEM model, this currently means only the `$PROBLEM` line in the new control stream will be updated.
-#' If `FALSE`, new model file will be an exact copy of its parent.
-#' @param .overwrite If `FALSE`, the default,  function will error if a model file already exists at specified `.new_model` path. If `TRUE` any existing file at `.new_model` will be overwritten silently.
-#' @param .directory Model directory which `.new_model` is relative to. Defaults to `options('rbabylon.model_directory')`, which can be set globally with `set_model_directory()`.
+#' @param .new_model Path to write new model files to **without file extension**.
```
Comment (seth127) [inline R/copy-model-from.R:12]: I'm a little annoyed we did this reformat. We kept the diff so clean otherwise.

File: R/submit-models.R [hunk 2]
```diff
@@ -0,0 +1,153 @@
+
```
Comment (seth127) [inline R/submit-models.R:1]: I feel the same way about this now. I'm gonna be glad we did it, but it does make the diff a little uglier.

File: tests/testthat/test-submit-model.R [hunk 3]
```diff
@@ -47,82 +39,8 @@ withr::with_options(list(rbabylon.model_directory = NULL), {
               })
             })
 
-
-  test_that("submit_model(.dry_run=T) with .ctl input parses correctly",
```
Comment (seth127) [inline tests/testthat/test-submit-model.R:51]: oof, so glad to see these go.

**Review by seth127 — COMMENTED** [review]
I can't approve my own changes but... these all look good to me.

File: R/copy-model-from.R (hunk 1 above)
Comment (copernican) [reply R/copy-model-from.R:12]: On the plus side, it's easier to read (no horizontal scrolling).

**Review by copernican — APPROVED** [review]
LGTM

**Pull request merged**